app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -52,6 +52,8 @@ def perform(form526_submission_id, full_name)
       end
     end
 
+    class AccountNotFoundError < RuntimeError; end
```

Comment:
It looks like the vast majority of custom exceptions in this repo subclass `StandardError` rather than `RuntimeError`. I don't have a strong preference.

Reply:
Given what you said and reading https://www.honeybadger.io/blog/ruby-exception-vs-standarderror-whats-the-difference/ and https://www.honeybadger.io/blog/ruby-custom-exceptions/, I'll change it to `StandardError`.